### PR TITLE
Captures new behaviour of the single node plan

### PR DESCRIPTION
**Pull request by vlad-stoian**
- Add new screenshot for both single node/cluster plans
- Reword `reachable_nodes` metrics explanation
- Modified the ODB RMQ architecture image to say `cf create-service`
instead of `cf create-instance`

Signed-off-by: Vlad Stoian <vstoian@pivotal.io>

### CODE REVIEW

**Comment by ghanna2017** [comment]
Thanks @vlad-stoian

File: install-config.html.md.erb [hunk 1]
```diff
@@ -86,15 +86,20 @@ In order to enable the on-demand service plan(s), you must configure the  **Dedi
 <p class="note"><strong>Note</strong>: If the ODB feature is not enabled, the ODB is deployed alongside the RabbitMQ installation, but it is not available in the Marketplace.</p>
 
 
-1. Choose the dedicated service instance you want to configure, Single Node Plan and/or Cluster Plan:
+1. Choose the dedicated service instance you want to configure:
+- #### For the single node plan:
```
Comment (vlad-stoian) [inline install-config.html.md.erb]: Hey @ghanna2017, for some reason, this line didn't get the markdown translation we would have hoped for. Do you mind deleting the bullet point and "####" and just making this a strong subheader? Same for the cluster plan below.

**Comment by ghanna2017** [comment]
Already fixed! :-)

**Comment by ghanna2017** [comment]
@vlad-stoian haven't pushed yet though... still editing your PR.

**Pull request merged**